csgo_bot.py

```diff
@@ -96,6 +96,24 @@ def list_mix(message):
 def test(message):
     print("TEST")
 
+def get_javigon(message):
+    chat_id = message.chat.id
+    bot.send_audio(chat_id=chat_id, audio=open('javigon(by ZEN).ogg', 'rb'))
+
+def get_zen(message):
+    chat_id = message.chat.id
+    message = "ZEN esta en la MIX"
+    bot.send_message(chat_id, message)
+    #in_mix(zen)
+    user = 'ZeN88'
+    if chat_id == GROUP_ID:
+        alias = "@" + user.username if user.username is not None else user.first_name
```

Comment:
Is this working?

Reply:
As long as alejando does not provide us the data base , i ve got no clue.

Reply:
I mean, you are defining user as a string in line 108 and then you are trying to access to properties of that string that will cause an error

```python
>>> user = 'pondere'
>>> user.user
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'str' object has no attribute 'user'

```

Reply:
solved